controls/mysql_conf.rb

```diff
@@ -55,12 +55,19 @@
   process_name = 'mariadbd' if os[:release] >= '9'
   service_name = 'mysqld'
   service_name = 'mariadb' if os[:release] >= '7'
+when 'suse'
+  mysql_config_path = '/etc/'
+  mysql_config_file = "#{mysql_config_path}my.cnf"
+  mysql_log_group = 'mysql'
+  process_name = 'mysqld'
+  service_name = 'mariadb'
 end
 
 control 'mysql-conf-01' do
   impact 0.5
   title 'ensure the service is enabled and running'
   describe service(service_name) do
+    p service_name
```

Comment:
this seems like an accidental change (?)

Reply:
fixed.